content/events/t1/repeat/compcafe.md

```diff
@@ -10,11 +10,11 @@ end_time = "21:00"
 location = "CS0.06"
 
 icon = "ph-coffee"
-colour = "academic"
+colour = "compcafe"
```

Comment:
We've had comp cafe be academic since stardut was created so i think leave it as that, especially when its the most frequent acadmeic event

Reply:
It's changed to match the insta schedule colouring. Also nice to separate compcafe from actual academic events